docker-compose.yml

```diff
@@ -1,4 +1,18 @@
 services:
   todo:
     build: .
-    network_mode: host
\ No newline at end of file
+    ports:
+      - 8080:8080
+    depends_on:
+      - database
+
+  database:
+    image: 'postgres:16.1-alpine3.19'
+    ports:
+      - 5432:5432
```

Comment:
@JoelBradley the port, was it meant to mismatch with the one in `postgres.rs`?

Reply:
(my server failed to start before)

Reply:
I think that it goes like this:

(port external to docker):(port internal to docker)

So if you want to connect to the database from something within this docker compose you'd use port 5432, but if you want to connect from outside you'd use 15432. I think it's useful to prevent port collisions on the host machine while keeping the default port inside docker. But for us it doesn't really matter.